dlc_developer_config.toml

```diff
@@ -34,11 +34,11 @@ deep_canary_mode = false
 [build]
```

Comment:
We might need to run this test as well: https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/ec2/test_efa.py#L152 (test_pytorch_efa_healthcheck)

Reply:
updated PR